client/src/components/organizations/OrganizationCard.vue

```diff
@@ -1,120 +1,215 @@
+        <ion-text class="orga-card-login__email body">{{ device.humanHandle.label }}</ion-text>
+        <div
+          v-show="!isDeviceLoggedIn(device)"
+          class="orga-card-login-time"
         >
-          <span>{{ device.organizationId?.substring(0, 2) }}</span>
-        </ion-avatar>
-        <ion-text
-          v-if="expirationDuration"
-          class="orga-expiration button-small"
-          :class="{ expired: isExpired(expirationDuration) }"
+          <ion-icon
+            :icon="time"
+            class="orga-card-login-time__icon"
+          />
+          <ion-text class="body-sm">
+            {{
+              device.deviceId in storedDeviceDataDict
+                ? $msTranslate(formatTimeSince(storedDeviceDataDict[device.deviceId].lastLogin, '--'))
+                : '--'
+            }}
+          </ion-text>
+        </div>
+        <div
+          v-show="isDeviceLoggedIn(device)"
+          class="orga-card-login-time connected"
         >
-          {{ $msTranslate(formatExpirationTime(expirationDuration)) }}
-        </ion-text>
-        <div class="orga-text">
-          <ion-card-title class="card-title">
-            <span class="title-h4">{{ device.organizationId }}</span>
-            <span class="subtitles-sm">{{ device.humanHandle.label }}</span>
-          </ion-card-title>
+          <ion-icon
+            :icon="ellipse"
+            class="success"
+          />
+          <ion-text class="body-sm">{{ $msTranslate('HomePage.organizationList.loggedIn') }}</ion-text>
         </div>
       </div>
-    </ion-card-header>
-  </ion-card>
+      <!-- trial expiration badge -->
+      <ion-text
+        v-if="expirationDuration"
+        class="orga-card-expiration button-small"
+        :class="{ expired: isExpired(expirationDuration) }"
+      >
+        {{ $msTranslate(formatExpirationTime(expirationDuration)) }}
+      </ion-text>
+    </ion-card-content>
+  </div>
 </template>
 
 <script setup lang="ts">
-import { AvailableDevice } from '@/parsec';
+import { AvailableDevice, isDeviceLoggedIn } from '@/parsec';
 import { getServerTypeFromHost, ServerType } from '@/services/parsecServers';
-import { IonAvatar, IonCard, IonCardHeader, IonCardTitle, IonText } from '@ionic/vue';
+import { StoredDeviceData } from '@/services/storageManager';
+import { IonAvatar, IonCardTitle, IonText, IonIcon, IonCardContent } from '@ionic/vue';
 import { onMounted, ref } from 'vue';
+import { ellipse, time } from 'ionicons/icons';
+import { MsImage, LogoIconGradient, formatTimeSince } from 'megashark-lib';
+import { formatExpirationTime, isExpired, getDurationBeforeExpiration } from '@/common/organization';
 import { Duration } from 'luxon';
-import { getDurationBeforeExpiration, formatExpirationTime, isExpired } from '@/common/organization';
+
+const isTrialOrg = ref(false);
+const expirationDuration = ref<Duration | undefined>(undefined);
+const storedDeviceDataDict = ref<{ [deviceId: string]: StoredDeviceData }>({});
```

Comment:
Not initialized. But you could instead pass the `lastLogin` as a prop from the OrganizationList, so you don't have to query the stored device data.